src/Symfony/Component/Translation/Translatable/TranslatableCurrency.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Translatable;
+
+use Money\Money;
+use Symfony\Contracts\Translation\TranslatableInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * Wrapper around PHP NumberFormatter for currency
+ * The provided currency is used instead of the locale's currency
+ *
+ * @author Sylvain Fabre <syl.fabre@gmail.com>
+ */
+class TranslatableCurrency implements TranslatableInterface
+{
+    private $value;
+    private $currency;
```

Comment:
should it be possible to define a specific locale as well?

also value+currency=money, no? In that sense perhaps TranslatableMoney is the better name ... we can also argue if the moneyphp shorthand is _that_ valueable, looking at it :)

Reply:
@ro0NL I don't understand what you meant by a specific locale. The locale comes from the Translator: why would you need another one here?

Reply:
Using a different locale for message arguments doesn't make sense IMO.